packages/xrpl/src/models/transactions/AMMInstanceCreate.ts

```diff
@@ -0,0 +1,76 @@
+import { ValidationError } from '../../errors'
+import { Amount } from '../common'
+
+import { BaseTransaction, isAmount, validateBaseTransaction } from './common'
+
+export const AMM_MAX_TRADING_FEE = 65000
+
+/**
+ * AMMInstanceCreate is used to create AccountRoot and the corresponding
+ * AMM ledger entries.
+ *
+ * This allows for the creation of only one AMM instance per unique asset pair.
+ */
+export interface AMMInstanceCreate extends BaseTransaction {
+  TransactionType: 'AMMInstanceCreate'
+
```

Comment:
There's an additional fee in the spec which is destroyed to create the AMMInstance, I'm not sure if it's on top of the normal `TransactionFee` or if it's the same thing since none of the other transactions listed have a `Fee` field. (Section 2.2.1) - Maybe this is a question for Aanchal?